syncer/syncer.go

```diff
@@ -843,18 +834,6 @@ func (s *Syncer) syncDDL(tctx *tcontext.Context, queueBucket string, db *DBConn,
 				err = s.handleSpecialDDLError(tctx, err, sqlJob.ddls, affected, db)
 				err = terror.WithScope(err, terror.ScopeDownstream)
 			}
-
-			if s.tracer.Enable() {
```

Comment:
don't support tracer any more?

Reply:
YES, tracer needs to be refactored in order to be compatible with the new shard DDL, and it has some more work need to do, I think we can refactor it later.